python/test_file_generator.py

```diff
@@ -2155,17 +2072,17 @@ def v681_2_file(self):
 		ts = self.ts_df.copy()
 		lar = self.lar_df.copy()
 		lar.action_taken = lar.action_taken.map(lambda x: random.choice(("4", "5", "6")))
-		lar.cltv = "15"
+		lar.cltv = "None"
```

Comment:
Is this value producing correct failures? 

A numeric value should produce the failures and pass the parser.

Reply:
The original value of 15 set the right error. Changing back to 15.